clients/src/main/java/org/apache/kafka/clients/GroupRebalanceConfig.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.kafka.clients;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.requests.JoinGroupRequest;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Class to extract group rebalance related configs.
+ */
+public class GroupRebalanceConfig {
```

Comment:
Should we call this something like `ConsumerGroupConfig`? All the configs here aren't directly linked to rebalancing